rfcs/RangeSlider.md

```diff
@@ -0,0 +1,62 @@
+# RFC: Add RangeSlider option to existing Slider component
+
+## Summary
+
+We are proposing to do some changes to the props of the existing Slider to expand its functionality. With the new option users will be able to use it as a RangeSlider.
+
+## Background
+
+The Azure Portal team is gradually embracing React for the development of Azure Portal, however there is no React-based UI library internally for Portal. So we decided to use FluentUI for basic UI needs. Whenever we cannot find a good match for an old portal control from existing FluentUI components, we would like to collaborate and contribute back to FluentUI to fill the gap.
+
+## Problem
+
+There is a control called RangeSlider that is used by a couple of Azure extension teams. (https://ms.portal.azure.com/?Microsoft_Azure_Playground=true#blade/Microsoft_Azure_Playground/ControlsIndexBlade/RangeSlider_createSimpleRangeSlider_Playground), which is not supported by current FluentUI. The current Slider component from FluentUI has only one thumb that allows the users to drag and change value. Since this kind of control is generic and no logic is specific to Portal, we propose to add features to the existing FluentUI slider, to make it capable of becoming a RangeSlider. So that the portal teams using RangeSlider will not be blocked when converting to ReactView, and FluentUI Slider component could provide more features.
+
+## Detailed Proposal
+
+Current RangeSlider control in Azure Portal is more complex than the existing fluent Slider component, in that it has markers below the slider bar and textboxes on both sides that allows user input\. To make things simple, the proposal only aims at making the minimum function working. We will focus on adding a thumb to make it a RangeSlider, and not talk about the other functionalities.
+
+### Details
```

Comment:
One additional thing to consider is how the keyboard and screen reader accessibility will work for this component. There's some authoring guidance for this pattern here: https://www.w3.org/TR/wai-aria-practices/#slidertwothumb

Reply:
Thanks for the guide. I already updated my PR based on the practices mentioned in the post.
https://github.com/microsoft/fluentui/pull/16854